packages/bolt-connection/src/bolt/bolt-protocol-v1.js

```diff
@@ -21,7 +21,7 @@ import {
   assertTxConfigIsEmpty,
   assertImpersonatedUserIsEmpty
 } from './bolt-protocol-util'
-import { Chunker } from '../chunking'
+import { Chunker } from '../channel/chunking'
```

Comment:
No such file `../chunking` - I think this is the right import.

Reply:
This was leftover of big refactoring, thanks for spotting.

Actually, it should be import from `../channel` directly, because we'd like to keep the modules only depending on the things exported by other module indexes. This way it's possible to reshape the internal of the modules without breaking the other ones.

Reply:
Makes sense. I've changed it to `../channel` now.